doc/developers_guide/developers_guide.md

```diff
@@ -2,19 +2,23 @@
 
 This guide contains information for developers.
 
+## Setup Integration Tests
+
+Integration tests need access to Google Cloud. To configure this, follow [these instructions](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating) to create a service account key and save it to file `google-key.json`.
+
 ## Running Regression Test
 
 This project contains some regression tests to monitor the performance cross releases. To run them locally use:
 
 ```shell
-mvn verify -P regressionTests
+mvn verify -PregressionTests
```

Comment:
Shouldn't this do the exact same thing?

Reply:
Good question. The workflow file uses `-PregressionTests`.